app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">JavaBaseProject</string>
-</resources>
\ No newline at end of file
+    <string name="parking_button">Button</string>
```

Comment:
this should be a more specific name like "activity_main_button_text"

Reply:
Done.